modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -258,6 +267,20 @@
     /** Prefix for snapshot threads. */
     public static final String SNAPSHOT_RUNNER_THREAD_PREFIX = "snapshot-runner";
 
+    /** Snapshot transfer rate distributed configuration key */
+    public static final String SNAPSHOT_TRANSFER_RATE_DMS_KEY = "snapshotTransferRate";
+
+    /** Snapshot transfer rate is unlimited by default. */
+    public static final long DFLT_SNAPSHOT_TRANSFER_RATE_BYTES = 0L;
+
+    /** Minimum block size for limited snapshot transfer. */
+    @SystemProperty(value = "Sets the minimum block size for limited snapshot transfer")
+    public static final String SNAPSHOT_LIMITED_TRANSFER_BLOCK_SIZE = "SNAPSHOT_LIMITED_TRANSFER_BLOCK_SIZE";
+
+    /** Snapshot limited transfer block size (in bytes). */
+    private static final int SNAPSHOT_TRANSFER_BLOCK_SIZE_BYTES =
```

Comment:
Why it's bounded to IO chunk size? Currently, this property is only used for local copy.

Reply:
System property removed.